maven-extension-rollout/rollout.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+commit_msg="Apply Gradle Enterprise Maven Extension"
+basedir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+repositories=$basedir/repositories.txt
+extensions_xml=$basedir/extensions.xml
+checkout_area=`mktemp -d`
+
+function process_repositories() {
+  num=`wc -l $repositories | awk '{ print $1 }'`
+  i=1
+  while read line; do
+  echo "($i/$num) Updating $line..."
+  process_repository $line
+  ((i++))
+  done < $repositories
+}
+
+function process_repository() {
+  repository_name="${1##*/}"
+  # clone the repository without actually downloading files or history
+  git clone -n $1 $checkout_area/$repository_name --depth 1 >& /dev/null
```

Comment:
I typically put all `$something` into quotes (`"$something"`) because this way it even works when the variable contains spaces.

Reply:
Done in d5fdb67